async_service/function_service/async_function_deployment.py

```diff
@@ -166,3 +167,32 @@ async def get_output(request_id: str, timeout: float = 2):
                 response_model=AsyncOutputResponse,
             )
         return app
+
+    def build_sync_server_app(self) -> FastAPI:
```

Comment:
I think we should include these routes as `/{name.lower()}/sync` in the same asgi app as `build_async_server_app`

maybe rename it to `build_publisher_app` to make it clearer